pindel/workflow.cwl

```diff
@@ -7,6 +7,7 @@ requirements:
     - class: ScatterFeatureRequirement
     - class: MultipleInputFeatureRequirement
     - class: SubworkflowFeatureRequirement
+    - class: InlineJavascriptRequirement
```

Comment:
Was this required to get the constant string to be recognized properly?  I'm still not 100% clear on when exactly we cross the line to needing Javascript.